amr-wind/wind_energy/actuator/disk/Joukowsky.H

```diff
@@ -37,6 +37,10 @@ struct JoukowskyData : public DiskBaseData
     vs::Vector disk_force{0.0, 0.0, 0.0};
     bool use_tip_correction{true};
     bool use_root_correction{true};
+    amrex::Real awc_amplitude{0.0};
+    amrex::Real awc_angular_frequency{0.0};
+    amrex::Real awc_azimuthal_mode{0.0};
```

Comment:
The azimuthal mode number should be an integer (Boundary layer flows allow for a real spanwise mode number, but for circular geometries it needs to be a signed integer).